packages/appstore/src/components/add-more-wallets/add-more-wallets.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text, Loading } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { useAvailableWallets } from '@deriv/hooks';
+import { observer } from '@deriv/stores';
+import CarouselContainer from './carousel-container';
+import AddWalletCard from './wallet-add-card';
+
+import './add-more-wallets.scss';
+
+const AddMoreWallets = observer(() => {
```

Comment:
Suppose we don't need observer here because we don't use useStore in this component